internal/integration/cli/etcd.go

```diff
@@ -39,6 +44,22 @@ func (suite *EtcdSuite) TestForfeitLeadership() {
 	)
 }
 
+// TestSnapshot tests etcd snapshot (backup).
+func (suite *EtcdSuite) TestSnapshot() {
```

Comment:
Is there a way to validate that the snapshot is valid? Like some primitive check that something can open it.

Reply:
it requires at least `etcdctl` :roll_eyes: 

once we have recovery flow, we'll combine both in the integration test, so we'll get some validation for it